### ISSUE TITLE
webjs call to get assets

### ISSUE THREAD

**Issue by Flaming19** [simolus3/web3dart#189]
Is there any way to get all assets from user address.

**Comment by VictorZhang2014** [comment]
Yes, it's by bscscan api or etherscan api to get all assets through an address. Normally, web3 lib can't help that.
